### ISSUE TITLE
UnableToResolveError: Unable to resolve module 

### ISSUE THREAD

**Issue by fazeelCGI** [reneweb/react-native-tensorflow#24]
I have installed the library and trying to use the available example. When I am loading the model from`../asset/tensorflow_inception_graph.pb` then it is giving me the error that unable to resolve the module. I have checked the path of model which is correct. Can anyone help to resolve this? error: bundling failed: UnableToResolveError: Unable to resolve module `../asset/tensorflow_inception_graph.pb` from `/Users/fazeel/CGapp/code/LoginDemo/src/screens/ImageRecognitionAI.js`: could not resolve `/Users/fazeel/CGapp/code/LoginDemo/src/asset/tensorflow_inception_graph.pb' as a file nor as a folder
    at ModuleResolver._loadAsFileOrDirOrThrow (/Users/fazeel/CGapp/code/LoginDemo/node_modules/metro-bundler/src/node-haste/DependencyGraph/ModuleResolution.js:337:11)

**Comment by reneweb** [comment]
It looks like it can't find the `tensorflow_inception_graph.pb` as a react native module. Have you made sure to add the `rn-cli.config.js` as described here: https://github.com/reneweb/react-native-tensorflow#fetching-files

**Comment by Pankajmalhan** [comment]
Yes!  I am getting the same error and I also add the "rn-cli.config.js" file in root directory of the project.

**Comment by SpQuyt** [comment]
https://stackoverflow.com/questions/50285949/unable-to-resolve-module-tensorflow-inception-graph-pb-as-a-file-nor-as-a-folder
This helped me a lot. You should watch this.

**Comment by vibss2397** [comment]
hey, i am having the same problem, i have dont everything, checked the paths, added the rn-cli.confing.js file, but still it doesnt seem to work
any help would be appreciated

**Comment by HarshitPKing** [comment]
hey, I am having the same problem, I have done everything, checked the paths, added the rn-cli.confing.js file, but still, it doesn't seem to work
any help would be appreciated

**Comment by chirag773** [comment]
hey I'm also having the same problem please help

**Comment by vibss2397** [comment]
React native has good documentation of how to integrate native modules , you need to have a base class which has all the functions, a module and a package file which act as a wrapper and integrate them into react native, check this out for an image classifier: 

https://github.com/vibss2397/See-Things/tree/master/android/app/src/main/java/com/seethings

This is the react native docs link:

https://facebook.github.io/react-native/docs/native-modules-android

**Comment by chirag773** [comment]
const text = {uri: './assets/tensorflow_labels.txt'}; 
      const graph = {uri: './assets/tensorflow_inception_graph.pb'}; 
      const tfImageRecognition = new TfImageRecognition({ 
        model: graph, 
        labels: text 
      });

this worked for me but now i'm gettiing an another error

Error:Could not load resource

**Comment by blindguydiy** [comment]
Have the same problem and also with other image classification examples.  I personnally think the people putting the examples up should give better explenations and how there examples work.  Also I think they should put examples on that work.
